apps/src/lib/util/mlApi.js

```diff
@@ -0,0 +1,27 @@
+/* global Promise */
+
+import $ from 'jquery';
+import {predict} from '@cdo/apps/MLTrainers';
+
+export const commands = {
+  async getPrediction(opts) {
+    return new Promise((resolve, reject) => {
+      $.ajax({
+        url: '/api/v1/ml_models/' + opts.modelId,
+        method: 'GET'
+      })
+        .then(modelData => {
+          const predictParams = {
+            ...modelData,
+            testData: opts.testValues
+          };
+          const result = predict(predictParams);
+          opts.callback(result);
+          return resolve();
+        })
+        .fail((jqXhr, status) => {
+          return reject({message: 'An error occurred'});
```

Comment:
Yeah, it does seem we might want to expose this error to the user's program, so they can then report it back to their end-user.